cmd/buildah/common.go

```diff
@@ -121,6 +116,20 @@ func getStore(c *cobra.Command) (storage.Store, error) {
 	return store, err
 }
 
+// setXDG sets XDG_RUNTIME_DIR when if it is unset under rootless
+func setXDG() error {
```

Comment:
There are other XDG variables, I'd prefer this to be named setXDGRuntimeDir() just to be clear which one we're setting.  But I won't lose sleep over it.  @rhatdan wdyt?

Otherwise LGTM.

Reply:
Much better, change the name of the function and then we can merge.